src/background/transactionBackground.ts

```diff
@@ -49,6 +49,7 @@ export default class TransactionBackground {
   public stopPolling = () => {
     if (this.getTransactionsInterval) {
       clearInterval(this.getTransactionsInterval);
+      this.getTransactionsInterval = undefined;
```

Comment:
Why does this fix it?

Reply:
clear interval only stops the interval from running, but this.getTransactionsInterval still has a number value.
In the startPolling, there is an if (!this.getTransactionsInterval) safety check to keep us from polling in two intervals, and while this.getTransactionsInterval is assigned to a number, it fails that check and will not restart the polling

Reply:
Oh it wasn't resetting when it left the page. Got it.